DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java

```diff
@@ -12,10 +12,12 @@
 import java.util.List;
 
 class PictureAdapter extends RecyclerView.Adapter<PictureAdapter.PictureViewHolder> {
+    private final Context mContext;
     private LayoutInflater mInflater;
     private List<PictureInfo> mPictureInfoList;
 
     PictureAdapter(Context context, List<PictureInfo> pictureInfoList) {
+        mContext = context;
```

Comment:
you are using the activity's Context, which is a bad practice. if you're really need it, use .getApplicationContext